.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
I know this is a template, but I don't see the `<!-- -->` notation for comments. I think having comments is better because it prevents template leaking into the issue if the user doesn't remove it.

Reply:
I notice the same thing in both files, but I will comment just once.

Reply:
I see what you mean. It was the default proposed by GitHub and it looked fine for now.
The change is already merged, you will have to propose a new PR.